components/org.wso2.carbon.identity.application.authenticator.fido/src/main/java/org/wso2/carbon/identity/application/authenticator/fido/util/FIDOAuthenticatorConstants.java

```diff
@@ -44,6 +44,8 @@ private FIDOAuthenticatorConstants() {
 
     public static final String WEBAUTHN_ENABLED = "FIDO.WebAuthn.Enable";
 
+    public static final String IS_USER_RESOLVED = "isUserResolved";
```

Comment:
check the possibility of getting this constant from a common constants class for all other authenticators